filebeat/docs/fields.asciidoc

```diff
@@ -4699,6 +4735,18 @@ type: keyword
 The name of the operating system.
 
 
+--
+
+*`traefik.access.user_agent.original`*::
++
+--
+type: text
```

Comment:
We should follow up in the docs to make sure non index fields don't have a type here as it could be confusing (not part of this PR).